nepalingo-web/src/hooks/useDictionary.tsx

```diff
@@ -0,0 +1,42 @@
+import useNewari from './useNewari'
+
+const Languages = [
+    'newari',
+    'tajpuriya'
+]
+
+export type DictionaryProps = {
+    language: typeof Languages[number],
+    word: string
+}
+
+export type DictionaryResponse = {
+    language: string;
+    word: string;
+    meanings: [{
+        audio?: { file: string, directory: string },
+        image?: string,
+        language: string,
+        meaning: string,
```

Comment:
rename to meaningOriginal?

naming convention camelCase
`meaningNP`, `meaningEN`